inbm/cloudadapter-agent/cloudadapter/client.py

```diff
@@ -38,15 +38,25 @@ def __init__(self) -> None:
         self._cloud_publisher = CloudPublisher(self._adapter)
 
     def _bind_agent_to_cloud(self) -> None:
-        """Bind Intel(R) In-Band Manageability messages to the cloud"""
-        self._broker.bind_callback(
-            TC_TOPIC.TELEMETRY,
-            lambda _, payload: self._cloud_publisher.publish_telemetry(payload)
-        )
-        self._broker.bind_callback(
-            TC_TOPIC.EVENT,
-            lambda _, payload: self._cloud_publisher.publish_event(payload)
-        )
+        ucc_flag = True     # Temporary-Remove once state file in place
```

Comment:
This needs to be changed now as state file is in place from installer.

Reply:
yep, thanks for the reminder.

Reply:
@gblewis1 - added.